compose/config/config.py

```diff
@@ -149,6 +162,32 @@ def find(base_dir, filenames):
         [ConfigFile.from_filename(f) for f in filenames])
 
 
+def get_config_version(config_details):
+    def get_version(config):
+        if config.config is None:
+            return 1
+        version = config.config.get('version', 1)
+        if isinstance(version, dict):
+            version = 1
```

Comment:
If the user for some reason has a service called version in a legacy compose file.